src/resdk/tables/qc_mappings.py

```diff
@@ -3,6 +3,14 @@
 Column mappings with the GENERAL_ prefix are designated for the general summary table in MultiQC.
 In this case, the mappings are not split by file from which the information is extracted
 but based on the type of information.
+
+If ''name'' key includes a list, it means that the column names have changed between MultiQC versions.
```

Comment:
Docstrings should be 79 or 99 characters per line. Here, each paragraph exceeds the limit by an increasing margin, making it look worse.

Reply:
Thank you for pointing this out. @marcellevstek could you prepare a fix PR?